mayhem/FuzzedDataProvider.h

```diff
@@ -0,0 +1,397 @@
+//===- FuzzedDataProvider.h - Utility header for fuzz targets ---*- C++ -* ===//
```

Comment:
Please don't include FDP, and just use the builtin version in the latest Clang.

Reply:
@xansec Done!